tools/klee-stats/klee-stats

```diff
@@ -229,7 +232,7 @@ def drawLineChart(vectors, titles):
 
     fig = plt.figure()
     for i in range(nFigs):
-        ax = fig.add_subplot(nRow, nCol, i)
+        ax = fig.add_subplot(nRow, nCol, i+1)
```

Comment:
Is this an unrelated bug?

Reply:
Yes, it's not related to this but I was not able to plot on my linux machines because of this (with matplotlib complaining that the number of plots had to be between 1 and N)